tasks/mesos.yml

```diff
@@ -35,6 +40,22 @@
   notify:
     - Restart mesos-slave
 
+- name: Mesos Slave attributes file
+  template: src=attributes-mesos-slave.j2 dest=/etc/mesos-slave/attributes
+  when:
```

Comment:
I'm not familiar with this `when` notation. Does it implies a `and` for all elements of the list?